src/themes/default/index.js

```diff
@@ -7,6 +7,12 @@ import '@vue-storefront/core/lib/passive-listeners'
 import { RouterManager } from '@vue-storefront/core/lib/router-manager'
 import { once } from '@vue-storefront/core/helpers'
 
+import { initCacheStorage } from '@vue-storefront/core/helpers/initCacheStorage';
+import { store as claimsStore } from 'theme/store/claims'
```

Comment:
I would only go for default export, it would be easier to have these lines here like:
```suggestion
import claimsStore from 'theme/store/claims'
```

Reply:
@patzick imho it's better to just rename named export so it's easier to find occurences but it's subjective change anyway

Reply:
@patzick  @filrak 
I have changed the names.